crates/hyperdrive-wrappers/build.rs

```diff
@@ -56,40 +58,41 @@ const TARGETS: &[&str] = &[
     "MockYieldSpaceMath",
 ];
 
-fn get_artifacts(artifacts_path: &Path) -> Result<Vec<(String, String)>> {
-    if !artifacts_path.exists() {
-        create_dir_all(artifacts_path)?;
-    }
-    let mut artifacts = Vec::new();
-    for entry in std::fs::read_dir(artifacts_path)? {
-        let entry = entry?;
-        let path = entry.path();
-        if path.is_file() {
-            let source = path.clone().into_os_string().into_string().unwrap();
-            let name = String::from(path.file_stem().unwrap().to_str().unwrap());
-
-            // If the artifact is one of our targets, add it to the list.
-            if TARGETS.contains(&name.as_str()) {
-                artifacts.push((source, name));
-            }
-        } else {
-            artifacts.extend(get_artifacts(&path)?);
-        }
-    }
-    Ok(artifacts)
-}
-
 fn main() -> Result<()> {
-    // Re-run this script whenever the build script itself or a contract changes.
+    // Re-run this script whenever .env, the build script itself, or a contract changes.
+    println!("cargo:rerun-if-changed=.env");
     println!("cargo:rerun-if-changed=build.rs");
     println!("cargo:rerun-if-changed=hyperdrive/contracts/");
```

Comment:
`println!("cargo:rerun-if-changed=hyperdrive/contracts/");`
is this going to work if the repo is not cloned beforehand?

Reply:
I think we can probably just remove this line